assets/css/style.css

```diff
@@ -115,6 +152,10 @@ a {
     text-align: center;
 }
 
+.about-me-title h2, .my-work-title h2, .my-contact-title h2 {
```

Comment:
consider placing this style within a class and adding it to the relevant elements - this will help to make your portfolio easily scalable in the future :-)

Reply:
thanks Dom